Src/Plugins/Security/SEC101_006.GitHubPatValidator.cs

```diff
@@ -18,6 +19,7 @@ public class GitHubPatValidator : ValidatorBase
         internal static GitHubPatValidator Instance;
 
         private const string PatExpression = "[0-9a-z]{40}";
+        private const string PatKey = "PATKEY";
```

Comment:
>private const string PatKey = "PATKEY"; [](start = 8, length = 39)

we could remove, since we are not using

Reply:
Removed.

---
In reply to: [575574928](https://github.com/microsoft/sarif-pattern-matcher/pull/189#discussion_r575574928) [](ancestors = 575574928)